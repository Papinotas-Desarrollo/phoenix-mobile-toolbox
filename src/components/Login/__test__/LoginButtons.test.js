import * as React from 'react';
import { TouchableOpacity } from 'react-native';
import { shallow } from 'enzyme';
import { LoginButtons } from '../';

describe('Snapshot test', () => {
  it('No buttons snapshot', () => {
    const buttons = [];
    const tree = shallow(<LoginButtons>{buttons}</LoginButtons>);
    expect(tree).toMatchSnapshot();
    expect(tree.children().length).toBe(0);
  });
  it('With buttons snapshot', () => {
    const buttons = [
      <TouchableOpacity key="1" />,
      <TouchableOpacity key="2" />,
    ];

    const tree = shallow(<LoginButtons>{buttons}</LoginButtons>);
    expect(tree).toMatchSnapshot();
    expect(tree.children().length).toBe(2);
  });
});
