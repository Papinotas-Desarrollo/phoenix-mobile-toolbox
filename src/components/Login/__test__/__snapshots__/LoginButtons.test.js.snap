// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test No buttons snapshot 1`] = `ShallowWrapper {}`;

exports[`Snapshot test With buttons snapshot 1`] = `ShallowWrapper {}`;
